internal/controller/gcpinstall/run.go

```diff
@@ -94,7 +174,7 @@ https://cloud.google.com/iam/docs/granting-changing-revoking-access`,
 		}),
 		command.NewWriteModel(command.Opts{
 			SkipIfSucceed: checkVMExists,
-			Command: `gcloud compute instances add-metadata ${VIRTUAL_MACHINE} \
+			Command: `gcloud compute instances get-metadata ${VIRTUAL_MACHINE} \
```

Comment:
Same here, why is this changing to `get` instead of `add`?

Reply:
:(((( idk -- reverted.